src/pages/index.tsx

```diff
@@ -22,6 +23,7 @@ const Index: FC = () => {
   useEffect(() => {
     if ('tasks' in response) {
       tasks = response.tasks;
+      tasks.sort((a, b) => (+a.endsOn - (+b.endsOn)));
```

Comment:
We need specify the type of params for a and b

Reply:
Specified the types of both a and b. 😁